doc/reference/reference_lua/box_read_view/list.rst

```diff
@@ -0,0 +1,42 @@
+.. _reference_lua-box_read_view_list:
+
+===============================================================================
+box.read_view.list()
+===============================================================================
+
+.. function:: read_view.list()
+
+    Returns an array of all active database read views.
+    This array might include the following read view types:
+
+    *   `read views <https://www.tarantool.io/en/enterprise_doc/read_views/>`_ created by application code (available in Tarantool Enterprise only)
+
+    *   system read views (used, for example, to make a :ref:`checkpoint <book_cfg_checkpoint_daemon>`
+        or join a new :ref:`replica <replication-architecture>`)
+
+    Read views created by application code also have the ``space`` field,
+    which lists all spaces available in a read view, and may be used like
+    a read view object returned by ``box.read_view.open()``.
```

Comment:
It also contains read views created with the "raw" C API (`box_raw_read_view_new()`). Those read views don't have a space list (like system read views, they can't be accessed from Lua).

Reply:
thanks, will fix